src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/helper/JwtAuthHelper.kt

```diff
@@ -4,15 +4,14 @@ import io.jsonwebtoken.Jwts
 import io.jsonwebtoken.SignatureAlgorithm
 import org.springframework.context.annotation.Bean
 import org.springframework.http.HttpHeaders
-import org.springframework.security.oauth2.jwt.NimbusReactiveJwtDecoder
-import org.springframework.security.oauth2.jwt.ReactiveJwtDecoder
+import org.springframework.security.oauth2.jwt.JwtDecoder
+import org.springframework.security.oauth2.jwt.NimbusJwtDecoder
 import org.springframework.stereotype.Component
 import java.security.KeyPair
 import java.security.KeyPairGenerator
 import java.security.interfaces.RSAPublicKey
 import java.time.Duration
-import java.util.Date
-import java.util.UUID
+import java.util.*
```

Comment:
i thought ktlint doesn't like star imports, but this seems to pass! i looked in incentives and couldn't see what was calling ktlint that perhaps _isn't_ calling it in this ci pipeline

Reply:
Hmmm, `gradle check` does call it